src/storagesync/azext_storagesync/_params.py

```diff
@@ -0,0 +1,159 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azext_storagesync._validators import parse_storage_account, parse_server_id
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type)
+
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+from knack.arguments import CLIArgumentType
+
+
+def load_arguments(self, _):
+    storage_sync_service_name_type = CLIArgumentType(help='The name of storage sync service.')
+    sync_group_name_type = CLIArgumentType(help='The name of sync group.')
+    cloud_endpoint_name_type = CLIArgumentType(help='The name of cloud endpoint.')
+    server_endpoint_name_type = CLIArgumentType(help='The name of server endpoint.')
+    azure_file_share_name_type = CLIArgumentType(help='The name of Azure file share.')
+    storage_account_type = CLIArgumentType(options_list='--storage-account',
+                                           help='The name or ID of the storage account.',
+                                           validator=parse_storage_account, id_part='name')
+    storage_account_tenant_id_type = CLIArgumentType(help='The id of the tenant that the storage account is in.')
+    server_resource_id_type = CLIArgumentType(options_list=['--registered-server-id', '--server-id'],
+                                              help='The resource id or GUID of the registered server.',
+                                              validator=parse_server_id)
+    server_id_type = CLIArgumentType(help='GUID identifying the on-premises server.')
+
+    with self.argument_context('storagesync storage-sync-service create') as c:
```

Comment:
The prefix of the subcommand name seems redundant. just use `storagesync service create`?

Reply:
After internal discussion, we decide to use 'storagesync create'

---
In reply to: [392831198](https://github.com/Azure/azure-cli-extensions/pull/1389#discussion_r392831198) [](ancestors = 392831198)